man/tpm2_duplicate.1.md

```diff
@@ -45,6 +51,8 @@ These options control the key importation process:
 
     Specifies the file path to the public key of the parent object on the
     destination TPM.  This should be a `TPM2B_PUBLIC` formatted file.
+    This public key is used for the wrapping of an openssl key
+    which will be exported for a remote TPM.
```

Comment:
An openssl key is too specific, it'll support any formatted PEM or DER key.
```markdown
This public key will be used for wrapping a plaintext key, eg PEM or DER, that is being duplicated to a new TPM storage parent.
See [Section 22 of TPM Spec 1.59](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part1_Architecture_pub.pdf) for further details.
```

Couldn't we also NOP -G for this option and figure it out? If it's an RSA key we use OAEP and if it's an ECC parent we perform that Diffe-Helman like process for the Z point.

Reply:
@williamcroberts @whooo I changed openssl key to PEM or DER key. Did I understand correctly that the PR can now be merged?

Reply:
> @williamcroberts @whooo I changed openssl key to PEM or DER key. Did I understand correctly that the PR can now be merged?

yes thank you